src/coreclr/scripts/superpmi-collect.proj

```diff
@@ -54,7 +59,7 @@
     <OutputMchPath>%HELIX_WORKITEM_UPLOAD_ROOT%</OutputMchPath>
     <!-- Workaround until https://github.com/dotnet/arcade/pull/6179 is not available -->
     <HelixResultsDestinationDir>$(BUILD_SOURCESDIRECTORY)\artifacts\helixresults</HelixResultsDestinationDir>
-    <WorkItemCommand>$(SuperPMIDirectory)\superpmi.py collect -log_level DEBUG --$(CollectionType) -pmi_location $(SuperPMIDirectory)\pmi.dll -pmi_path $(SuperPMIDirectory)\crossgen2</WorkItemCommand>
+    <WorkItemCommand>$(SuperPMIDirectory)\superpmi.py collect -log_level DEBUG --$(CollectionType) -pmi_location $(SuperPMIDirectory)\pmi.dll -pmi_path @(PmiPathDirectories->'%(FullPath)', ' ')</WorkItemCommand>
```

Comment:
this will expand to `-pmi_path <folder1> <folder2>`? I thought you wanted `-pmi_path folder1 -pmi_path folder2`?

Reply:
Yes, I was mistaken how nargs works in Python.
This needs to be expanded to `-pmi_path <folder1> <folder2>` - I found this while locally testing the change.